components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -84,11 +89,13 @@ export const BlurryPoster = ({
       <img
         src={getProcessedImage(bgImageSrc, '1200x1200', bgImageFocus)}
         alt={bgImageAlt || ''}
+        aria-describedby={caption ? 'story-hero-caption' : undefined}
```

Comment:
This is the background image in the hero (optional). It shares the same caption with the foreground hero image (optional)